README.md

```diff
@@ -1,18 +1,22 @@
 # plusminus
 
-The **plusminus** package provides a ready-to-run arithmetic parser and evaluator, based on `pyparsing`'s 
-`infixNotation` helper method.
+The **plusminus** package provides a ready-to-run arithmetic parser and evaluator, based on [`pyparsing`](https://github.com/pyparsing/pyparsing)'s 
+[`infixNotation`](https://github.com/pyparsing/pyparsing/blob/master/pyparsing/helpers.py#L608) helper method.
```

Comment:
Links to code by line are probably pretty fragile - better to link to docs at `pyparsing-doc.readthedocs`